### DISCUSSION TITLE
Mi pokedex =)

### DISCUSSION THREAD

**Discussion by hackit92** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#463] [Show and tell]
Que tal compañeros, solo quería compartirles mi práctica de pokedex, no tuve mucho tiempo por cuestiones de trabajo pero creo que cumple con la mayoría de los puntos, solo subí el sonido de los primeros 150 pokemones por lo que en el resto no suena, ojalá les guste =)

https://hackit92.github.io/PracticasJS/

**Comment by ErickMontesDK** [comment]
Muy buen detalle del audio. Quedó padre

**Reply by hackit92** [reply]
muchas gracias!

**Comment by mibarra24** [comment]
Gran trabajo compañero 👍

**Reply by hackit92** [reply]
Gracias!

**Comment by MaribelMendezM** [comment]
que bonito trabajo

**Reply by hackit92** [reply]
muchas gracias =)

**Comment by OscarPedroza** [comment]
Excelente trabajo compañero!

**Reply by hackit92** [reply]
Muchas gracias!
